frb_dart/lib/src/ffi.dart

```diff
@@ -8,6 +8,9 @@ export 'ffi/stub.dart'
 typedef DropFnType = void Function(PlatformPointer);
 typedef ShareFnType = PlatformPointer Function(PlatformPointer);
 
+/// Rust SyncReturn<usize> type is forced cast to u64.
+const pointerLength = 8;
```

Comment:
uber-nit: shall we rename it to sth like "pseudoPointerLength" etc, since when a future reader is reading code using this const, he may think "oh I know, pointerLength means 32bit on 32bit device and 64bit on 64bit device".